int8 id
int8 color
geometry_msgs/Pose pose